### DISCUSSION TITLE
Simplest form/type of simulation ?

### DISCUSSION THREAD

**Discussion by MrMemozz** [SpiceSharp/SpiceSharp#226] [Q&A] [answered]
I have been using a dc sweep to make sure that the unity integration and the UI/component placement is not affecting the simulation negatively.
 But if i were to use the simplest form of simulation, which would be as simple as after the user connects their circuit (sources, resistors, capacitors, MultiMeters ...etc) each with with its own Values, they click a button, and they get the results they want on their multimeters? basically which simulation type should i use ?
and would it work even with multiple Sources ?
i tried searching the documentation but i studied EE in a different language so it gets kind of messy for me with the different Terms.
Thank you!

**Comment by svenboulanger** [comment, marked as answer]
I'd probably start out with the `OP` simulation type. This computes the operating point of the circuit, and is the solution to the circuit if nothing would be varying in time. It is used as the starting point for most transient and frequency simulations as well.

**Reply by MrMemozz** [reply]
Thank you so much sven!
